source/_posts/azure-purview.md

```diff
@@ -0,0 +1,149 @@
+
+<img src="/images/azure-purview/create-resource.png" />
+
+The Basic tab is pretty straightforward. Provide a valid resource group and name for the Purview account and select the appropriate location suitable for your usage.
+
+<img src="/images/azure-purview/configuration.png" />
+
+In the configuration tab, there are a few new options. As Azure Purview is still in preview, not much is known about what the platform sizing, catalog options and/or data insights settings mean (the learn more button only points to the pricing). 
+
+But, knowing that Azure Purview focuses on doing scans, platform size might have to do with scanning frequency and the number of scans you can perform. 
+
+Under the **catalog** category, there are two options (which are greyed out due to it being in preview). Here you can select which one is relevant for your implementation. The two options as of writing this blog are:
+
+- Sources registration, automated scans and data discovery
+- Business glossary and lineage visualization
+
+The last category, **Data insights**, is about creating statistics and analytics for all the data that you scanned. 
+
+After selecting the correct settings, you can create the resource. 
+
+*Note: It can take quite a while to create this service.*
+
+## Azure Purview Studio
+
+The Azure Purview Studio is the overview on which you can start your scans, look at lineage add business glossary and much more.
+
+Azure Purview Studio contains a search field in which you can look through all the Assets that have been scanned. To get there, we first have to register some data sources and perform a scan. 
+
+<img src="/images/azure-purview/purview-studio.png" />
+
+### Register datasources
+
+To get started with registering datasources, either click on the "Register sources" pane or on the database icon on the left side.
+
+The first time you arrive on the sources page, there wont be any collections. Azure Purview uses collections to group different datasources. This gives you the opportunity to, for example, create a specific collection for each of the environments.
+
+Start out by creating a collection by clicking on "**+ new collection**". The collection requires you to fill in a name. In this example a collection called "westeuropedatasources" was created in which one could store all data sources available in the West Europe region. 
+
+<img src="/images/azure-purview/collection.png" />
+
+After a collection is created, you can start registering data sources. Currently, because of preview the number of supported data sources is limited. However, this list is sure to grow quickly and will also include multi –cloud data sources (i.e. giving you the ability to scan data in AWS) 
+
+<img src="/images/azure-purview/datasources.png" />
+
+For now, we will use the Azure SQL Database to scan. When selecting the Azure SQL Database to scan, it will give the the option to select a database, which will then be registered under sources. 
```

Comment:
the Azure SQL Database >> an Azure SQL Database data source
the Azure SQL >> an Azure SQL